src/components/ErrorState/ErrorState.jsx

```diff
@@ -0,0 +1,3 @@
+export const ErrorState = () => {
+  return <div style={{textAlign: "center"}}>Server Error</div>;

```

Comment:
Avoid using the `style` attribute as much as possible. Add a utility css class for this.